### PR TITLE
First commits

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request contains the first commits to create this Ansible role from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).

## 💭 Motivation and context ##

This Ansible role installs and configures the OS to trust some CA certificates required to operate with CISA VENOM.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
🐍 Sssssssstrong work! 🐍

File: tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,26 @@
 ---
-# tasks file for skeleton
+- name: Load var file specific to the OS type
+  include_vars: "{{ lookup('first_found', params) }}"
```
Comment (mcdonnnj) [inline tasks/main.yml]:
```suggestion
  ansible.builtin.include_vars: "{{ lookup('first_found', params) }}"
```

File: README.md [hunk 2]
```diff
@@ -4,13 +4,9 @@
...
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started.  This skeleton project contains
-[licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role.
+This Ansible role installs some CA certificates that are required for
+the VENOM agents to talk back to the VENOM mothership.  It also
+instructs the system to trust these certificates.
```
Comment (mcdonnnj) [inline README.md]: I think we should expand the acronym on the first usage.

```suggestion
the VENOM (Virtual Enterprise Network Operations Manager) agents to
talk back to the VENOM mothership.  It also instructs the system to
trust these certificates.
```

File: molecule/default/tests/test_default.py [hunk 3]
```diff
@@ -12,7 +12,29 @@
...
+        "cert_US_CISA_0_Root_CA.crt",
+        "cert_US_CISA_1_Issuing_CA_reissued.crt",
+    ],
+)
+def test_cert_files(host, file_name):
+    """Verify that all expected certificate files were installed."""
+    distribution = host.system_info.distribution
+    if distribution == "amzn" or distribution == "fedora":
+        path = "/etc/pki/ca-trust/source/anchors/"
+    elif distribution == "debian" or distribution == "kali" or distribution == "ubuntu":
```
Comment (mcdonnnj) [inline molecule/default/tests/test_default.py]: Would doing a distro in list check like below be a little less repetitive? Just a thought.

```python
distribution in ["distro1", "distro2"]:
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪💪💪 Just had a few thoughts and a suggestion.

File: molecule/default/tests/test_default.py (hunk 3 above)
Comment (jsf9k) [reply molecule/default/tests/test_default.py]: Excellent suggestion!  See commit cfa4e70.

**Pull request merged**